distributed/compatibility.py

```diff
@@ -1,11 +1,14 @@
+from __future__ import annotations
+
 import logging
 import platform
 import sys
 
 import tornado
 
-logging_names = logging._levelToName.copy()
-logging_names.update(logging._nameToLevel)
+logging_names: dict[str | int, int | str] = {}
+logging_names.update(logging._levelToName)  # type: ignore
+logging_names.update(logging._nameToLevel)  # type: ignore
```

Comment:
We're loosing a `copy()` here, just wanted to double check that's okay

Reply:
We're replacing a copy() with an update() for the sake of readability. There's no functional difference.